### ISSUE TITLE
France 	France 

### ISSUE THREAD

**Issue by DavidViral** [CSSEGISandData/COVID-19#691]
France in Province and Country field

**Comment by yetzt** [comment]
No, but France is split in different areas, such as their overseas territories, some French Guyana and – most important – the whole rest that is for simplicitys sake called France.

**Comment by JiPiBi** [comment]
If interested for France  : see #705
